src/components/TeamCarousel.tsx

```diff
@@ -0,0 +1,74 @@
+"use client";
+
+import useEmblaCarousel from "embla-carousel-react";
+import { useCallback, useState } from "react";
+import CarouselItem from "~/components/CarouselItem";
+import AutoScroll from "embla-carousel-auto-scroll";
+import members from "~/assets/footer/members.json";
+
+export default function Carousel() {
+  const [hoveredIdx, setHoveredIdx] = useState<number | null>(null);
+
+  const [emblaRef, emblaApi] = useEmblaCarousel(
+    {
+      loop: true,
+    },
+    [AutoScroll({ stopOnMouseEnter: true, startDelay: 0, speed: 0.8 })],
+  );
+
+  const handleCarouselStop = useCallback(() => {
+    emblaApi?.plugins().autoScroll?.stop();
+  }, [emblaApi]);
+
+  const handleCarouselResume = useCallback(() => {
+    emblaApi?.plugins().autoScroll?.play();
+  }, [emblaApi]);
+
+  const handleImageActive = useCallback((idx: number) => {
+    setHoveredIdx(idx);
+  }, []);
+
+  const handleImageInactive = useCallback(() => {
+    setHoveredIdx(null);
+  }, []);
+
+  return (
+    <>
+      <div className="relative w-full overflow-hidden pb-12">
+        {hoveredIdx !== null && members[hoveredIdx] && (
+          <div className="absolute bottom-2/12 left-1/2 -translate-x-1/2 text-center text-xs font-semibold">
```

Comment:
Please do not use absolute positioning for this, use the layout of the DOM to implement this (but make sure there is no layout shift on hover)